src/components/app-bar/components/search-button/search-button.stories.tsx

```diff
@@ -0,0 +1,10 @@
+import { Meta, StoryObj } from '@storybook/react'
+import { NavSearchButton } from './search-button'
+
+export default {
+  title: 'Components/App bar/Search button',
+  component: NavSearchButton,
+} as Meta<typeof NavSearchButton>
+
+export const StylesOnlyUsage: StoryObj<typeof NavSearchButton> = {}
+export const ReactUsage: StoryObj<typeof NavSearchButton> = {}
```

Comment:
this component is really simple, but since it doesn't have a bigger consumer to give an example I just created this stories